frontend/public/components/utils/headings.tsx

```diff
@@ -119,7 +119,10 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
           {badge && <SplitItem>{badge}</SplitItem>}
         </Split>
       )}
-      <h1 className={classNames('co-m-pane__heading', { 'co-m-pane__heading--logo': props.icon })}>
+      <h1
+        className={classNames('co-m-pane__heading', { 'co-m-pane__heading--logo': props.icon })}
+        style={resourceTitle ? {} : { margin: 0 }}
```

Comment:
There's potentially many other things in this component that still requires the margin and you cannot simply check for presence of `resourceTitle`.
You should probably just omit rendering the `h1` altogether if there's no contents to render.

Reply:
Added a check & reverted previous change.